### ISSUE TITLE
Semantic checker needs to be integrated into SIF and GraphML importers

### ISSUE THREAD

**Issue by dondi** [dondi/GRNsight#386]
The SIF and GraphML import routes (_import-controller.js_) do not run their output through the semantic checker. This should be done so that our semantic checks our consistent across all formats. The best way to integrate this is to refactor the import helper function in _import-controller.js_.

**Comment by kdahlquist** [comment]
Assigning this to @mihirsamdarshi and bumping it up to priority 0.

**Comment by kdahlquist** [comment]
@mihirsamdarshi is working on this right now.  Says will be done by next week.

**Comment by kdahlquist** [comment]
@ebachoura and @eileenchoe are in fact working on this exact issue because they thought that this issue was a sub-issue of #362 / #363. @mihirsamdarshi needs talk to them.

**Comment by kdahlquist** [comment]
@ebachoura and @eileenchoe, I need clarification as to the status of this one.  Is it done? or is there more work needed?

**Comment by eileenchoe** [comment]
@mihirsamdarshi is reviewing the semantic checker integration for SIF an GraphML and will update status when finished

**Comment by mihirsamdarshi** [comment]
reviewed, it works

**Comment by kdahlquist** [comment]
@dondi has not had time to review this one, so I did some quick checks on what I think should be semantic errors.  
* I made a .sif file with a node name that was too long; that brought up the error as expected.
* I made a .sif file with a duplicated row, which should trigger the duplicate gene name error, but it still loaded.  So, for whatever reason, this is not being caught by the semantic checker.  I'm attaching the file I made here.
[duplicate-gene-name.zip](https://github.com/dondi/GRNsight/files/945554/duplicate-gene-name.zip)

To thoroughly check this, we probably need to try making the .sif and .graphml versions of an assortment of .xlsx test files and load 'em up and see what happens.
